lib/getConfigForFile.js

```diff
@@ -4,16 +4,18 @@ const augmentConfigFull = require("./augmentConfig").augmentConfigFull
 const configurationError = require("./utils/configurationError")
 const path = require("path")
 
+/*:: type returned$config = Promise<?{ config: stylelint$config, filepath: string }>*/
+
 module.exports = function (
   stylelint/*: stylelint$internalApi*/,
```

Comment:
A pretty strange situation with this annotation. If I take the definition of the `stylelint$internalApi` and place it in this file explicitly, coverage will rise up to +2/+3%... But at the same time, as you can see on [snapshot,](http://pix.toile-libre.org/upload/original/1488926821.png) it says that it expects `string`  but not the `Object`